vendor/user_profile.sh.default

```diff
@@ -0,0 +1,32 @@
+# use this file to run your own startup commands for msys2 bash'
+
+# To add a new vendor to the path, do something like:
+# export PATH=${CMDER_ROOT}/vendor/whatever:${PATH}
+
+function launch_ssh_agent {
```

Comment:
A better place for this might be a file in `${CMDER_ROOT}/vendor/bin` that could be `.` sourced after being uncommentd in `${CMDER_ROOT}/config/user_profile.sh` that was created from `${CMDER_ROOT}/vendor/user_profile.sh.default`.

That way we control it going forward as our code, anything in `${CMDER_ROOT}/config` belongs to the user and we will create it if it does not exist but we never touch it again.

Reply:
Good call, I hadn't considered that. Done in 7e1bc42